python/pyproject.toml

```diff
@@ -55,3 +57,38 @@ highlight_language = "none"
 html_theme = "pydata_sphinx_theme"
 html_logo = "../images/logo.png"
 html_show_copyright = false
+
+[tool.pytest.ini_options]
+minversion = "7.0"
+addopts = "-v --cache-clear -rf --doctest-modules"
+console_output_style = "count"
+
+[tool.tox]
+legacy_tox_ini = """
+[tox]
+min_version = 4.0
+envlist =
+    py{39, 310, 311}-lowest
```

Comment:
We end the `lowest-direct` tests at Python 3.11 since [`myst-parser` didn't add support for 3.12 until 3.0](https://myst-parser.readthedocs.io/en/latest/develop/_changelog.html#id3).